models/reporting/olids/person_status/dim_person_residence.sql

```
@@ -0,0 +1,42 @@
+        base.person_id,
+        base.sk_patient_id,
+        base.uprn_hash,
+        frailty.frail_flag
+    from base
+    left join frailty on base.person_id = frailty.person_id
+),
+
+household_size as (
+    select
+        uprn_hash,
+        count (*) as household_registered_count,
```

SQLFluff (4.2.2)

[error] Function name not immediately followed by parenthesis.

(LT06)